CMakeLists.txt

```diff
@@ -845,5 +872,15 @@ cpack_ifw_configure_component(OpenStudioApp
   REQUIRES_ADMIN_RIGHTS
 )
 
-include(CPack)
+cpack_add_component(Libraries
+  DISPLAY_NAME "Install required system libraries"
+  DESCRIPTION  "Install compiler-provided system runtime libraries, and Windows Universal CRT libraries for app-local deployment"
+  REQUIRED
+  HIDDEN
+)
 
+message("CMAKE_CPACK_COMMAND=${CMAKE_CPACK_COMMAND}")
+add_custom_target(PACKAGE_DEBUG
+  COMMAND ${CMAKE_CPACK_COMMAND} --debug --verbose --config CPackConfig.cmake
+  COMMENTS "Build package with debug and verbose output"
+)
```

Comment:
Add a debug target for the package. Won't work for regular MSVC I think, you need $(Configuration) in there (Release/Debug)